fabfile.py

```diff
@@ -81,6 +86,10 @@ def update():
     run('systemctl restart uwsgi.service')
     run('systemctl restart nginx')
 
+    # Update secrets
```

Comment:
what's that?

Reply:
I thought of updating secrets (eg. if you want to just update secrets, not reinstall the whole env.). Maybe it would not be necessary.